package/doc/sphinx/source/documentation_pages/analysis_modules.rst

```diff
@@ -19,7 +19,7 @@ has to import them from :mod:`MDAnalysis.analysis`, for instance ::
 
     import MDAnalysis.analysis.align
 
-.. Note:: 
```

Comment:
The diffusionmap.rst will need an `automodule` command or something, copy one of the existing ones in there